### ISSUE TITLE
Bank Approval Prediction: Evaluate and perform a comparitive study across various machine learning models. Suggest why certain algorithms might be doing better than the others.

### ISSUE THREAD

**Issue by akash-kamalesh** [dscpesu/SimpleML#7]
---

| Field  | Description                       |
| ------ | --------------------------------- |
| About  | A short Description about project |
| Github | Your Github name                  |
| Email  |                                   |
| Label  | Update request                    |

---

**Define You**

- [ ] Hacktober Fest Contributor

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like...**

**Describe alternatives you've considered?**

**Approach to be followed (optional):**

**Additional context**

**Comment by Lekhya25** [comment]
I would like to be assigned to this task.

Since this is a classification problem, we can use the following models:
KNN Classifier
Logistic Regressor
ANN
Random Forest Classifier
Naive Bayes
Decision Tree
Stochastic Gradient Descent
Support Vector Machines

To compare the models, we can use metrics like Confusion matrix, F1 score, ROC-AUC etc.
We can finally do hyperparameter tuning etc on the best model to further improve the model performance and accuracy.
